ml-algorithms/build.gradle

```diff
@@ -51,9 +51,8 @@ dependencies {
     implementation("ai.djl.onnxruntime:onnxruntime-engine:0.21.0") {
         exclude group: "com.microsoft.onnxruntime", module: "onnxruntime"
     }
-    def os = DefaultNativePlatform.currentOperatingSystem
-    //mac doesn't support GPU
-    if (os.macOsX) {
+    //arm doesn't support GPU
+    if (System.getProperty("os.arch") == "aarch64") {
```

Comment:
To make sure why can't we keep both then like:

`if os.macOsX || System.getProperty("os.arch") == "aarch64"`

Reply:
that makes a lot of sense.